### DISCUSSION TITLE
Make New Mod Selection Menu Horizontally Scrollable When Holding CTRL Key

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#18293] [Ideas] [answered]
Scrolling horizontally can be done with scrolling outside the selection boxes already, but adding ability to ignoring boxes and scrolling horizontally when holding CTRL would be better in my opinion.

**Comment by peppy** [comment]
Just to check, are you aware that you can drag-scroll horizontally?

**Reply by ghost** [reply]
Yes, but it is necessary to scroll by holding down and dragging with the mouse1 or scrolling with the normal mouse wheel by coming to the scrollbar at the bottom.

**Comment by peppy** [comment, marked as answer]
The generally accepted key for this is shift, not control (macOS and windows). I believe we should support this at the framework side.

**Reply by peppy** [reply]
... and shift already works on both macOS and windows. It is implemented at an OS level and handled (almost) correctly by osu! already. The only issue is that it's reverse direction from what it should be. This is being tracked at https://github.com/ppy/osu-framework/issues/5167.

**Reply by ghost** [reply]
Ah yes, I missdeemed Shift with CTRL, sorry.
